pkg/interceptors/cel/cel_test.go

```diff
@@ -202,3 +231,61 @@ func TestFilterEvaluation(t *testing.T) {
 		})
 	}
 }
+
+func TestExpressionEvaluation(t *testing.T) {
```

Comment:
I like this focused method for testing expressions! Would we be able to also include another tests for errors - e.g. mismatched types, invalid expressions, etc?